api/lib/domain/usecases/update-user-details-for-administration.js

```diff
@@ -7,7 +9,26 @@ module.exports = async function updateUserDetailsForAdministration({
     await userRepository.isEmailAllowedToUseForCurrentUser(userId, userDetailsForAdministration.email);
```

Comment:
**remarque HORS SCOPE**
Pareil, ici je trouve bizarre d'avoir un repo qui commence par `is` ... ça veut dire qu'on y cache la règle métier qui permet de savoir si on autorise un email pour un certain utilisateur ?
En lisant cette ligne, comme je ne connais pas le scope, je ne saurai pas dire si on doit autoriser un utilisateur à utiliser un email ou non. Alors que dans l'idéal je pense que j'aurai dû pouvoir lire cette info dans le usecase... 🤔

Reply:
Il est vrai que je suis parti de cet existant, que j'ai copié son format pour l'adapter au `username`.
Mais, je ne sais pas pourquoi il a été écrit comme cela.
Le retour attendu par une méthode commençant par ìs` serait  un booléen.

Reply:
Suite au changement apporté sur le repo, la fonction s'appelle désormais `findAnotherUserByEmail`